src/notation/notationmodule.cpp

```diff
@@ -187,7 +187,10 @@ void NotationModule::registerUiTypes()
     qmlRegisterType<UndoRedoToolbarModel>("MuseScore.NotationScene", 1, 0, "UndoRedoToolbarModel");
     qmlRegisterType<UndoHistoryModel>("MuseScore.NotationScene", 1, 0, "UndoHistoryModel");
     qmlRegisterType<TimelineView>("MuseScore.NotationScene", 1, 0, "TimelineView");
+
+    qmlRegisterUncreatableType<SelectionFilterSection>("MuseScore.NotationScene", 1, 0, "SelectionFilterSection", "Cannot create");
```

Comment:
I find it slightly scary that there is now a QML type and a C++ type with this name. But as long as it works, and there are really no warnings printed at runtime, I guess it's fine.